### DISCUSSION TITLE
Welcome to ecutweaker Discussions!

### DISCUSSION THREAD

**Discussion by cedricp** [cedricp/ecutweaker#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hichemkingsman** [comment]
je souhaite debloqué l'application ecutweaker

**Reply by Furtif** [reply]
elle n'est pas bloqué!!!

**Comment by SpagurdiCat** [comment]
So i stumbled upon this program, how do you use it on a Windows 10 computer? Is it only made for Android OS? Im new to github so i have trouble navigating through categories.

**Reply by Furtif** [reply]
Windows Mac Linux uses pc version https://github.com/cedricp/ddt4all

**Comment by hknozclk** [comment]
Hello,
I want to write the injector code on a 2005 1.5dci Kangoo vehicle. I want to do this using the ECU Tweaker APK. Is this possible? I would appreciate your help
